test/extended/images/helper.go

```diff
@@ -24,8 +24,8 @@ func GetImageLabels(c client.ImageStreamImageInterface, imageRepoName, imageRef
 }
 
 // ModifySourceCode will modify source code in the pod of the application
-// according to the sed script.
-func ModifySourceCode(oc *exutil.CLI, selector labels.Selector, sedScript, file string) error {
+// according to the provided command.
+func ModifySourceCode(oc *exutil.CLI, selector labels.Selector, cmd []string) error {
```

Comment:
In the ruby extended test I actually am removing `public/index.html` file with that function. Why it's not appropriate ? Deleting file is a modification as well.